src/main/java/com/limechain/babe/dto/EpochSlot.java

```diff
@@ -0,0 +1,18 @@
+package com.limechain.babe.dto;
+
+import lombok.Value;
+
+import java.io.Serializable;
+import java.math.BigInteger;
+import java.time.Duration;
+import java.time.Instant;
+
+@Value
+public class EpochSlot implements Serializable {
```

Comment:
Can you consider choosing another name for this class because it's misleading whether it provides information for an epoch or a slot?

Reply:
The idea behind this structure is to represent information only related to a babe/epoch slot. If we take into account that a slot is only related to BABE and nothing else we might as well just call it `Slot`.